app/models/manageiq/providers/base_manager.rb

```diff
@@ -17,5 +19,11 @@ def ext_management_system
     def refresher
       self.class::Refresher
     end
+
+    private
+
+    def require_credentials
+      errors.add(:authentications, 'You must enter credentials.') if missing_credentials?
```

Comment:
A methods in EmsCloudController does not add "empty" authentication to EMS. In case credentials were not filled, EMS gets no empty authentication, but none. In that case validates_associated does not fail.
I am thinking about adjusting controller method with a transient attribute on ems and checking it in custom validation, will update PR.